src/ORM/FieldType/DBTime.php

```diff
@@ -26,33 +27,30 @@ class DBTime extends DBField
 {
     /**
      * Standard ISO format string for time in CLDR standard format
+     * This is equivalent to php date format "H:i:s" e.g. 09:30:00
      */
     public const ISO_TIME = 'HH:mm:ss';
 
+    private static array $field_validators = [
+        TimeFieldValidator::class,
+    ];
+
     public function setValue(mixed $value, null|array|ModelData $record = null, bool $markChanged = true): static
     {
         $value = $this->parseTime($value);
-        if ($value === false) {
-            throw new InvalidArgumentException(
-                'Invalid date passed. Use ' . $this->getISOFormat() . ' to prevent this error.'
-            );
-        }
         $this->value = $value;
```

Comment:
```suggestion
```

Reply:
Similar to DBDate, I've reverted this class to how it was in CMS 5 and have made the minimal required changes for this PR e.g. field_validators config